app/models/flat.rb

```diff
@@ -1,3 +1,9 @@
 class Flat < ApplicationRecord
   belongs_to :user
+  has_many :reservations
+  has_many :users, through: :reservations
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :description, presence: true
+  validates :price, presence: true
```

Comment:
Does this need: , numericality: { only_integer: true }

Reply:
?

Reply:
ok cool!

Reply:
i think it should actually be numericality: true so it can be a float